packages/react-icons-mdl2-branded/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@fluentui/react-icons-mdl2-branded",
+  "version": "1.0.0-beta.0",
+  "description": "Branded SVG icons from the MDL2 icon set",
+  "private": false,
+  "main": "lib-commonjs/index.js",
+  "module": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "sideEffects": [
+    "lib/version.js"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluentui"
+  },
+  "license": "MIT",
+  "scripts": {
+    "build": "just-scripts build",
+    "bundle": "just-scripts bundle",
+    "clean": "just-scripts clean",
+    "code-style": "just-scripts code-style",
+    "just": "just-scripts",
+    "lint": "just-scripts lint",
+    "start": "just-scripts dev:storybook"
+  },
+  "devDependencies": {
+    "@fluentui/eslint-plugin": "^1.0.0-beta.0",
+    "@types/react": "16.9.42",
+    "@types/react-dom": "16.9.10",
+    "@types/webpack-env": "1.16.0",
+    "@fluentui/scripts": "^1.0.0",
+    "react": "16.8.6",
+    "react-app-polyfill": "~1.0.1",
+    "react-dom": "16.8.6"
+  },
+  "dependencies": {
+    "@fluentui/set-version": "^8.0.0-beta.0",
+    "@fluentui/react-icons-mdl2": "^1.0.0-beta.11",
```

Comment:
Be sure and update both of these to the latest versions just before check-in.

Same with eslint-plugin probably.